gluetool/pylint/option_default.py

```diff
@@ -0,0 +1,134 @@
+"""
+Checker for using unknown module options.
+
+Does not handle multiple gluetool modules within the same Python module well, all their
+options are stored under the same key (file name), which leads to false negatives. This
+will be fixed later.
+"""
+
+from pylint.checkers import BaseChecker, utils
+from pylint.interfaces import IAstroidChecker
+
+
+BASE_ID = 76
```

Comment:
It'd deserve even to be moved into a `__init__.py` file, since it's common for all gluetool's checkers, however, in the simplest form, that leads to a circular dependency. So, I want to get rid of its repetition, adding a docstring, but that will take a bit more work to achieve.